README.md

```diff
@@ -92,6 +92,27 @@ Pylance provides users with the ability to customize their Python language suppo
         - `true`
         - `false`
 
+-   `python.analysis.autoImportCompletions`
+
+    -   Used to control the offering of auto-imports in completions.
+    -   Accepted values:
+        -   `true` (default)
+        -   `false`
+
+-   `python.analysis.packageIndexDepths`
+
+    -   Used to control how many installed packages indexer will index.
+    -   Default value: empty array
+    -   Accepted values: 
+        -   Tuple of [```package name```, ```depth to scan```, ```include all symbols```]
```

Comment:
What happens when `include all symbols` is set to false?

Reply:
we only include symbols in ```__all__```, if include all symbol is ```true```, we index every symbol decl in the file.

Reply:
Can we clarify that here? If it's too complex to explain here, maybe point the user to the detailed section below?

Reply:
We could do `include all symbols?` to reflect it's a bool, and then add one more bullet point under the accepted values explaining "If `include all symbols` is set to false, only symbols in `__all__` are included. When it's set to `true`, Pylance will index every module/top level symbol declarations in the file".